src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
+@import "./utils/variables.scss";
+
 body {
   margin: 0;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+
+  width: $calendar-width;
+  padding: $calendar-padding;
+
+  &__day {
+    box-sizing: border-box;
+    cursor: pointer;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $square-side;
+    height: $square-side;
+    border: 1px solid #000;
+    background-color: #eee;
+
+    transition: transform 0.5s;
```

Comment:
You can add a transition to a background-color too, it will be more smooth.

Reply:
Mind @BudnikOleksii comment - check the syntax of `transition` - https://stackoverflow.com/questions/7048313/how-to-have-multiple-css-transitions-on-an-element

Reply:
yep. Thank you for a useful link